cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -325,11 +325,11 @@ def test_service_ansible_playbook_order_credentials(ansible_catalog_item, ansibl
             "machine_credential": ansible_credential.name
         }
     view = navigate_to(service_catalog, "Order")
-    wait_for(lambda: view.machine_credential.is_displayed, timeout=5)
+    view.wait_displayed()
```

Comment:
I was not entirely sure about .wait_displayed() and (view.fields('credential')).visible_widget and please let me know if there is a better way to accomplish this.